test/integration/ferc1_eia_train_test.py

```diff
@@ -105,47 +82,62 @@ def eia_ferc1_training_data():
             ["2_2020_plant_owned_195", "3_2020_plant_owned_195"],
             ["f1_steam_2020_12_2_0_1", "f1_steam_2020_12_2_0_1"],
             [18, 18],
-            marks=pytest.mark.xfail(
-                reason="Duplicate FERC1 ids in overrides", raises=AssertionError
-            ),
+            pytest.raises(AssertionError, match=r"Found record_id_ferc1 duplicates"),
         ),
         pytest.param(
             ["True"],
             [2020],
             ["299_2020_plant_total_14354"],  # EIA id already in training data
             ["f1_steam_2020_12_134_2_2"],  # FERC1 id NOT in training data
             [246],
-            marks=pytest.mark.xfail(reason="EIA id already in training data."),
+            pytest.raises(
+                AssertionError,
+                match=r"The following EIA records area already in the training data",
```

Comment:
```suggestion
                match=r"The following EIA records are already in the training data",
```

Looks like the other instances of this string are in `pudl.analysis.record_linkage.eia_ferc1_train.py:589` and `:598`.